### PR TITLE
invert the colors of steps in campus and qualification forms

### LINKED ISSUE #270: Invert the colours for the active step
For both the campus and qualification forms.

Current colours are not clear enough to make it obvious which step you are on.

Screenshot 2019-04-23 at 22 58 11

### PR DESCRIPTION
**Pull request by boney-bun**
fix #270

### CODE REVIEW

**Comment by boney-bun** [comment]
PIA:
ford3#276

File: django_project/ford3/static/css/wizard_form.css [hunk 1]
```diff
@@ -34,18 +34,22 @@
 }
 
 .nav-link.done {
-    color: #7ed321 !important;
-    border: 1px solid #7ed321;
+    color: #7dccd0 !important;
```
Comment (pierrealixt) [inline django_project/ford3/static/css/wizard_form.css]: Can you use CSS variable? See `stylesheet.css`

**Comment by boney-bun** [comment]
@pierrealixt i've changed the color into using defined color (i named it: light blue) in stylesheet.
i think, in the future, we will need to store all defined colors in the css into variables for consistency.

**Review by Christiaanvdm — APPROVED** [review]
LGTM

**Pull request merged**